e2e_test/source/basic/kafka.slt

```diff
@@ -668,12 +680,27 @@ order by
 56166    1
 56166    2
 
+query I 
+SELECT * FROM s24;
```

Comment:
maybe consider moving this part to `e2e_test/batch` since it is conducting a batch query